flake.nix

```diff
@@ -4,12 +4,14 @@
   inputs.nixpkgs.url = "github:NixOS/nixpkgs/nixos-22.11";
   # TODO: https://github.com/aws/aws-lc/pull/830
   inputs.awslc.url = "github:dougch/aws-lc?ref=nixv1.36.0";
+  inputs.awslc-fips.url = "github:dougch/aws-lc?ref=nixfips-2024-09-27";
 
-  outputs = { self, nix, nixpkgs, awslc, flake-utils }:
+  outputs = { self, nix, nixpkgs, awslc, awslc-fips, flake-utils }:
     flake-utils.lib.eachDefaultSystem (system:
       let
         pkgs = nixpkgs.legacyPackages.${system};
         aws-lc = awslc.packages.${system}.aws-lc;
+        aws-lc-fips = awslc-fips.packages.${system}.aws-lc-fips;
         # TODO: submit a flake PR
```

Comment:
Which version of awslc-fips is this? The one our CI calls "awslc-fips" or the one it calls "awslc-fips-2022"? Or is this a completely different version?

Reply:
PR says it's [fips-2024-09-27](https://github.com/aws/aws-lc/tree/fips-2024-09-27)

Reply:
And I'm guessing that's not the "awslc-fips" used by the rest of the CI? Is the inconsistency a potential problem? Which version of awslc-fips SHOULD we be testing with?

Reply:
Yeah, the rest of the CI is 2022. My preference is to test with the newest version though. Not sure why we're able to upgrade our nix awslc version easier than the rest of the CI.